src/main/resources/com/xpandit/plugins/xrayjenkins/task/XrayImportBuilder/config.jelly

```diff
@@ -44,6 +44,24 @@
             </div>
         </f:entry>
     </j:file>
+
+    <j:file xmlns:j="jelly:core" var="addSwitchableTestField" omitXmlDeclaration='true' escapeText='false' trim='true'
+    		outputMode='HTML'>
+    	<f:entry title="{{label}}">
+    		<select name="_.inputTestInfoSwitcher" class="setting-input"
```

Comment:
This option value is not being saved. Steps to recreate:
1) Open a Build and add a Import Results task
2) Change the value of the "data type" for the Test Info part to "JSON Content"
3) Save
4) Return to the previous build
5) Expected: This select should display "JSON Content" by instead returned to it's default value ("File Path")